src/app/about/page.tsx

```
@@ -0,0 +1,204 @@
+              "/best.webp",
+              "/bgmi.jpg",
+              "/team2.jpg",
+            ].map((src, index) => (
+              <CarouselItem
+                key={index}
+                className="flex h-[300px] w-full items-center justify-center transition-transform duration-500 ease-in-out sm:h-[400px] md:h-[450px]"
+              >
+                <img
+                  src={src}
+                  alt={`Image ${index + 1}`}
+                  className="h-full w-full rounded-lg object-cover"
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)